src/plugins/intel_cpu/src/config.cpp

```diff
@@ -305,45 +364,47 @@ void Config::updateProperties() {
     using namespace InferenceEngine;
```

Comment:
`InferenceEngine` namespace not used implicitly now, may be changed to `ov::threading`

Reply:
It seems some legacy code still use it, this PR didn't completely remove legacy code. Anyhow, I will double check this file.

Reply:
You are right, we don't need it any more. It has been removed.